scripts/start_local.sh

```diff
@@ -24,11 +25,22 @@ if [ -z "$ARGO_CLOUDOPS_GIT_AUTH_METHOD" ]; then
     export ARGO_CLOUDOPS_GIT_AUTH_METHOD=https
 fi
 
-# TODO / HACK: Vault was not loading credentials from the default chain.
+# Vault was not loading credentials from the default chain, try to fetch from profile
 if [ -n "${AWS_PROFILE}" ]; then
-    export AWS_ACCESS_KEY_ID=`aws configure get $AWS_PROFILE.aws_access_key_id`
-    export AWS_SECRET_ACCESS_KEY=`aws configure get $AWS_PROFILE.aws_secret_access_key`
-    export AWS_SESSION_TOKEN=`aws configure get $AWS_PROFILE.aws_session_token`
+    CREDS_PROCESS_VALUE=`aws configure get $AWS_PROFILE.credential_process`
+    if [ -n "$CREDS_PROCESS_VALUE" ]; then
+        # profile is using credential_process - parse json from it
+        CREDS_JSON=`$CREDS_PROCESS_VALUE`
+        export AWS_ACCESS_KEY_ID=`echo "$CREDS_JSON" | jq -r ."AccessKeyId"`
```

Comment:
We should probably check that jq exists on the system before executing it. You may be able to use the aws --query command to parse the json instead of relying on another binary.